conversions/hexadecimal_to_octal2.c

```diff
@@ -0,0 +1,119 @@
+/**
+ * @file
+ * @brief Convert hexadecimal number to octal number (with decimal intermediary)
+ * @details
+ * The input is valid from 0 to 0xFFFF_FFFF_FFFF_FFFF.
+ *
+ * At first, this program converts a hex string to an unsigned long long
+ * decimal, and then to an octal string.
+ *
+ * When there is an invalid character in input string, this program stops
+ * parsing and converts the string until that character.
+ *
+ * @see hexadecimal_to_octal.c
+ */
+
+#include <stdio.h>
+#include <string.h>
```

Comment:
Add a brief description of what the header/library does (what does it add).
```suggestion
#include <stdio.h>  /// for printf() and scanf()
#include <string.h> ///
```

Reply:
Applied at df1b119.